themes/bootstrap3/js/facets.js

```diff
@@ -1,5 +1,87 @@
 /*global VuFind */
 /*exported initFacetTree */
+
+/* --- Facet List --- */
+VuFind.register('facetList', function FacetList() {
+  function overrideHref(selector, overrideParams = {}) {
+    $(selector).each(function overrideHrefEach() {
+      let dummyDomain = 'https://www.example.org'; // we need this since the URL class cannot parse relative URLs
+      let url = new URL(dummyDomain + VuFind.path + $(this).attr('href'));
+      Object.entries(overrideParams).forEach(([key, value]) => {
+        url.searchParams.set(key, value);
+      });
+      url = url.href;
+      url = url.replaceAll(dummyDomain, '');
+      $(this).attr('href', url);
+    });
+  }
+
+  function updateHrefContains() {
+    let overrideParams = { contains: $('.ajax_param[data-name="contains"]').val() };
+    overrideHref('.js-facet-sort', overrideParams);
+    overrideHref('.js-facet-next-page', overrideParams);
+    overrideHref('.js-facet-prev-page', overrideParams);
+  }
+
+  function getContent(overrideParams = {}) {
```

Comment:
Would it make sense to display some kind of spinner or loading message while content is being fetched? Otherwise, there's no feedback between the time the user stops typing and the time results display. If this takes a second or two, it can be a bit disconcerting.

Reply:
You're right! I added this to updateContent(), since getContent() is just the bare AJAX functionality.